src/cli/arguments.md

```diff
@@ -1,3 +1,5 @@
+# Clap basic
```

Comment:
So this file needs to have these lines removed.  Move the {{#include..}} from ansi_terminal.md to this file